specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2018-08-20-preview/healthcare-apis.json

```diff
@@ -0,0 +1,882 @@
+            "schema": {
+              "$ref": "#/definitions/OperationResultsDescription"
+            }
+          },
+          "404": {
+            "description": "No operation result was found matching operationResultId.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          },
+          "default": {
+            "description": "Default error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "ServicesProperties": {
+      "description": "The properties of a service instance.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state.",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Creating",
+            "Accepted",
+            "Verifying",
+            "Updating",
+            "Failed",
+            "Canceled",
+            "Deprovisioned"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        },
+        "accessControlSettings": {
+          "$ref": "#/definitions/ServiceAccessControlSettingsInfo",
+          "description": "The access control settings of the service instance."
+        },
+        "cosmosDbSettings": {
+          "$ref": "#/definitions/ServiceCosmosDbSettingsInfo",
+          "description": "The settings for the Cosmos DB database backing the service."
+        }
+      },
+      "required": [
+        "accessControlSettings"
+      ]
+    },
+    "ServiceAccessControlSettingsInfo": {
+      "description": "The access control settings of the service instance.",
+      "type": "object",
+      "properties": {
+        "authority": {
+          "description": "Url to the authentication authority.",
+          "type": "string"
+        },
+        "allowedOids": {
+          "description": "List of object IDs that are allowed to access the FHIR service.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "authority",
+        "allowedOids"
+      ]
+    },
+    "ServiceCosmosDbSettingsInfo": {
+      "description": "The settings for the Cosmos DB database backing the service.",
+      "type": "object",
+      "properties": {
+        "offerThroughput": {
+          "type": "integer",
+          "description": "The provisioned throughput for the backing database.",
+          "readOnly": false
+        }
+      }
+    },
+    "ServicesSkuInfo": {
+      "description": "Information about the SKU of the service instance.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the SKU.",
+          "enum": [
+            "Standard"
```

Comment:
Your resource doesn't have to have a sku, do you plan on adding more values in the future?